applications/plugins/SofaCUDA/sofa/gpu/cuda/CudaMechanicalObject.h

```diff
@@ -187,26 +187,6 @@ using sofa::gpu::cuda::CudaVec3f1Types;
 using sofa::gpu::cuda::CudaVec6fTypes;
 using sofa::gpu::cuda::CudaRigid3fTypes;
 
-
-// template specialization must be in the same namespace as original namespace for GCC 4.1
-// g++ 4.1 requires template instantiations to be declared on a parent namespace from the template class.
-extern template class SOFA_GPU_CUDA_API sofa::component::container::MechanicalObject<CudaVec1fTypes>;
```

Comment:
extern template are important to reduce compilation cost. I see no reason why we should remove that.